api-tester/src/main/java/com/revenuecat/apitester/java/SubscriptionOptionsAPI.java

```diff
@@ -0,0 +1,25 @@
+package com.revenuecat.apitester.java;
+
+import com.revenuecat.purchases.models.PricingPhase;
+import com.revenuecat.purchases.models.SubscriptionOption;
+import com.revenuecat.purchases.models.SubscriptionOptions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@SuppressWarnings({"unused"})
+final class SubscriptionOptionsAPI {
+
+    static void checkSubscriptionOptions() {
+        SubscriptionOptions subscriptionOptions = new SubscriptionOptions(new ArrayList());
```

Comment:
maybe we should make the constructor private? or is there a use case for the customer creating them

Reply:
Don't think there is! But... this couldn't be `private` because modules 😔

Reply:
Ah yeah yeah, this has to be public because its used in the Google module in `storeProductConverstions.kt`

Reply:
gah, the modules are starting to get frustrating. i'd say we can just let this go and re-visit when we think more about the modules.... though it would be nice to have a list of things we want to potentially refactor like this one, could we add to the ticket?